README.md

```diff
@@ -22,148 +19,183 @@ Table of contents
    * [Jenkins pipeline](#jenkins-pipeline)
    * [Contribution guide](#contribution-guide)
    * [License](#license)
-<!--te-->
-
-## Development quick start
-Built with REACT.js, Bootstrap 4, Babel, Workbox, and Webpack.
-### Prerequisites:
-If you haven’t already, install the following software:
-* Install [Git](https://git-scm.com/downloads).
-* Install [Node.js](https://nodejs.org/en/download/).
-* Install [Visual Studio Code](https://code.visualstudio.com/) from Microsoft and the following extensions:<br/>
+
+## Elastic Path REACT PWA Reference Storefront
+
+### Documentation Introduction
+
+This document provides guidelines for knowledgeable JavaScript developers and front-end developers to extend and customize React Reference Storefront components. However, this document is not a primer for JavaScript or CSS and is intended for professionals who are familiar with the following technologies:
+  * [React](https://reactjs.org/)
+  * [jQuery](https://jquery.com/)
+  * [Bootstrap](https://getbootstrap.com/docs/4.0/getting-started/introduction/)
+  * [CSS](https://en.wikipedia.org/wiki/Cascading_Style_Sheets)
+  * [less](http://lesscss.org/)
+
+### Related Resources
+
+[Elastic Path REACT PWA Reference Storefront Overview](https://developers.elasticpath.com/reference-experiences)
+[Elastic Path REACT PWA Reference Storefront Installation Guide](https://elasticpath.github.io/react-pwa-reference-storefront/)
+[Requirements and Specifications](https://elasticpath.github.io/react-pwa-reference-storefront/documentation/technologyoverview/)
+
+## Overview
+
+The Elastic Path REACT PWA Reference Storefront is a flexible e-commerce website built on Elastic Path’s RESTful e-commerce API, Cortex API.Through Cortex API, the storefront uses the e-commerce capabilities provided by Elastic Path Commerce and gets data in a RESTful manner. For more information about the storefront, see [Elastic Path REACT PWA Reference Storefront documentation](https://elasticpath.github.io/react-pwa-reference-storefront/).
```

Comment:
This is correct, but is it all necessary for the technical docs in the README? I'm cool with it staying if needed